### ISSUE TITLE
`auto changelog --quiet` should output raw changelog

### ISSUE THREAD

**Issue by laughedelic** [intuit/auto#1869]
**Describe the bug**

`auto changelog --dry-run --quiet` should output only the changelog.

**To Reproduce**

If I run

```shell
$ auto changelog --dry-run
ℹ  info      Potential Changelog Addition:
 #### ⚠️ Pushed to `main`

- some changes ([@laughedelic](https://github.com/laughedelic))

#### Authors: 1

- Alexey Alekhin ([@laughedelic](https://github.com/laughedelic))
$
```

I get the changelog, but also some logging (`ℹ  info`). I want to get the same, but without the logs:

```
$ auto changelog --dry-run --quiet
$
```

output is empty.

**Expected behavior**

From the help:

```
-q, --quiet            Print **only** the result of the command
```

So I expect that the result of the `changelog` command is the generated changelog text.

**Environment information:**

I don't think it's environment specific, but anyway:

<details>
<summary>auto info</summary>

```txt

Environment Information:

"auto" version: v10.18.2
"git"  version: v2.30.2
"node" version: v12.18.1

Project Information:

✔ Repository:      laughedelic/auto-release-test (​https://github.com/laughedelic/auto-release-test​)
✔ Author Name:     Alexey Alekhin
✔ Author Email:    alexey.alekhin@tapad.com
✔ Current Version: v0.0.10
✔ Latest Release:  v0.0.10 (​https://github.com/laughedelic/auto-release-test/releases/tag/v0.0.10​)

✔ Labels configured on GitHub project 

GitHub Token Information:

✔ Token:            [Token starting with ec6e]
✔ Repo Permission:  admin
✔ User:             laughedelic
✔ API:              undefined (​undefined​)
✔ Enabled Scopes:   repo
✔ Rate Limit:       4943/5000
```

</details>

**Additional context**

I wasn't sure if this should be filed as a bug, a question or a feature request. Sorry if I missed 😅 

I got the idea to use `--quiet` flag from this answer: https://github.com/intuit/auto/discussions/1867#discussioncomment-472335

**Comment by adierkens** [comment]
:rocket: Issue was released in [`v10.18.8`](https://github.com/intuit/auto/releases/tag/v10.18.8) :rocket:

**Comment by laughedelic** [comment]
Wow! that was quick! ⚡ thanks!
